lib/matplotlib/_mathtext.py

```diff
@@ -2841,6 +2857,38 @@ def binom(self, s, loc, toks):
         return self._genfrac('(', ')', 0.0, self._MathStyle.TEXTSTYLE,
                              num, den)
 
+    def _genset(self, state, body, annotation, overunder):
+        thickness = state.font_output.get_underline_thickness(
+            state.font, state.fontsize, state.dpi)
+
+        body.shrink()
+
+        cbody = HCentered([body])
+        cannotation = HCentered([annotation])
+        width = max(cbody.width, cannotation.width)
+        cbody.hpack(width, 'exactly')
+        cannotation.hpack(width, 'exactly')
+
+        kern_vgap = thickness * 3
+        if overunder == "under":
+            vlist = Vlist([cannotation,                # annotation
+                           Kern(kern_vgap),            # space
+                           cbody                       # body
+                           ])
+            # Shift so the annotation sits in the same vertical position
+            shift_amount = cannotation.depth + cbody.height + kern_vgap
+
+            vlist.shift_amount = shift_amount
+        else:
+            vlist = Vlist([cbody,                      # body
+                           Kern(kern_vgap),            # space
+                           cannotation                 # annotation
+                           ])
+
+        hlist = [Hlist([vlist])]
```

Comment:
Do you need to wrap the return value in a list containing an Hlist?  Can you not just return the Vlist directly?  That's what e.g. c_over_c does.
(Again, I don't know the correct answer...)

Reply:
For simple purposes of readability, and if we ever wanted there to be a horizontal gap between the current symbol and the next one - we'd use:
```python
hlist = [Hlist([vlist, Hbox(0, horizontal_gap)])]
```

Reply:
I'd rather not wrap things into more things unless needed; if you want I guess you can leave the note about horizontal gaps as a comment?

Reply:
I made the change, although, I found other places where this is done too:
https://github.com/matplotlib/matplotlib/blob/0520035717925131666ad5bb2ef7e21ec61b40fd/lib/matplotlib/_mathtext.py#L2684-L2687

Reply:
I guess these can be cleaned up as a separate PR?